map/map_tf_generator/src/map_tf_generator_node.cpp

```diff
@@ -51,19 +55,21 @@ class MapTFGeneratorNode : public rclcpp::Node
 
   void onPointCloud(const sensor_msgs::msg::PointCloud2::ConstSharedPtr clouds_ros)
   {
+    srand(3939);
```

Comment:
Could you add a comment like this?

```suggestion
    // Fix random seed to ensure the same results every time
    srand(3939);
```

Reply:
Resolved in 93384d477bc67b6dbeea6b90a285a75b35b7f46a